### PR TITLE
added home page

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Added Navigation bar 
-> Added Carousel in home page.
-> Made the page responsive .

### CODE REVIEW

File: src/Components/Sliders/Sliders.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useState, useEffect, useRef } from "react";
+import "./Sliders.css";
+const sources = [
```
Comment (shubhamkumar648) [inline src/Components/Sliders/Sliders.jsx:4]: i like the way you keep img link in array

**Review by shubhamkumar648 — APPROVED** [review]
code structures  and functionality is good , good to go

File: src/Components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  {
+    name: "Playlist",
+    icon_class: "fas fa-play",
+  },
+  {
+    name: "History",
+    icon_class: "fas fa-history",
+  },
+];
```
Comment (tushartiwari7) [inline src/Components/Navbar/Navbar.jsx:25]: This is nice

**Review by tushartiwari7 — APPROVED** [review]
Code is very neat and clean.
Really liked the carousel functionality you did.
Go ahead and merge this.

**Pull request merged**